client/src/pages/Feed.js

```diff
@@ -108,16 +280,60 @@ const Feed = () => {
         handleFilterModal,
         handleQuit,
         handleLocation,
+        handleOnClose,
+        showFilters,
       }}
     >
       <FeedWrapper>
-        <FilterBox />
-        <Posts filteredPosts={fakePosts} />
-        <SvgIcon
-          src={creatPost}
-          onClick={handleCreatePost}
-          className="create-post"
-        />
+        <LayoutWrapper>
+          <SiderWrapper
+            breakpoint="md"
+            className="site-layout-background"
+            width={290}>
+            <div>
+              <MenuWrapper
+                defaultSelectedKeys={['ALL']}
+                onClick={handleChangeType}
+              >
+                {Object.keys(HELP_TYPE).map((item, index) =>
+                  <Menu.Item key={item}>
+                    {HELP_TYPE[item]}
+                  </Menu.Item>
+                )}
+              </MenuWrapper>
+              <FiltersWrapper>
+                <button
+                  onClick={handleShowFilters} >
+                  <span>
+                    <FiltersIcon />
+                  </span>
+                Filters
```

Comment:
Let's give it a color black

current
<img width="282" alt="Screen Shot 2020-05-10 at 8 29 38 PM" src="https://user-images.githubusercontent.com/42228386/81521746-f9e26880-92fc-11ea-9a1b-04fe944382b3.png">

figma
<img width="241" alt="Screen Shot 2020-05-10 at 8 29 28 PM" src="https://user-images.githubusercontent.com/42228386/81521741-f3ec8780-92fc-11ea-821f-34297f9c744b.png">

Reply:
Done.